src/Categories.jsx

```diff
@@ -0,0 +1,10 @@
+import { useSelector } from 'react-redux';
+
+export default function Categories({ categories, onClick }) {
+  const { selectedIdx } = useSelector((state) => ({ selectedIdx: state.selectedIdx }));
+  return (
+    <ul>
+      {categories.map((category) => (<li key={category.id}><button type="button" onClick={() => onClick(category.id)}>{selectedIdx === category.id ? `${category.name}(V)` : category.name}</button></li>))}
```

Comment:
너무 길어지면 줄바꿈을 해주세요 ;)
```suggestion
      {categories.map((category) => (
        <li key={category.id}>
          <button 
            type="button" 
            onClick={() => onClick(category.id)}
          >
            {selectedIdx === category.id ? `${category.name}(V)` : category.name}
          </button>
        </li>
      ))}
```